### DISCUSSION TITLE
Simple formatting example

### DISCUSSION THREAD

**Discussion by Quafadas** [outr/scribe#321] [Q&A] [answered]
```
import scribe.output._

...

      scribe.info(
        out(
          white("two")
        )
      )
```
And I'm getting this compile error... 
```
[error]     |  Found:    scribe.output.LogOutput
[error]     |  Required: scribe.message.LoggableMessage  
```
Woudl anyone have a hint on what I might be missing?

**Comment by darkfrog26** [comment]
What version of Scribe are you using?

**Reply by Quafadas** [reply]
```val scribeVersion = "3.10.1"``` ... 

```
val scala = "3.1.3"
val scalajs = "1.10.0"
```

**Comment by darkfrog26** [comment, marked as answer]
That's a bug I accidentally introduced in `3.1`. Good catch! This is resolved in `3.10.2` which was just released. Thanks!

**Reply by Quafadas** [reply]
:-)... thanks for looking!
